src/app/custom/translations.tsv

```diff
@@ -722,31 +722,37 @@
-"ws_post-v1-addresses-parse_title"	"Address breakdown"	"Décomposition d'une adresse"
-"ws_post-v1-addresses-parse_description"	"Split a text address into several fields."	"Découpe une adresse au format texte en plusieurs champs."
-"ws_post-v1-detect-lang_title"	"Detection of the language of a text"	"Détection de la langue d’un texte"
+"ws_post-v1-detect-lang_title"	"langDetect - Detection of the language of a text"	"langDetect - Détection de la langue d’un texte"
 "ws_post-v1-detect-lang_description"	"Detects the language of a text document."	"Détecte la langue d’un document texte."
-"ws_post-v1-en-classify_title"	"Classification into scientific fields Pascal-Francis"	"Classification en domaines scientifiques Pascal-Francis "
+"ws_post-v1-en-classify_title"	"pascalFrancisClass - Classification into scientific fields Pascal-Francis"	"pascalFrancisClass - Classification en domaines scientifiques Pascal-Francis "
 "ws_post-v1-en-classify_description"	"Automatically classifies scientific documents in English using the Pascal (Science, Technology and Medicine) or Francis (Humanities and Social Sciences) classification schemes."	"Classe automatiquement des documents scientifiques en anglais dans le plan de classement Pascal (Sciences, Techniques et Médecine) ou Francis (Sciences Humaines et Sociales)."
-"ws_post-v1-en-classhalen_title"	"Classification in HAL domains"	"Classification dans les domaines HAL"
+"ws_post-v1-en-classhalen_title"	"halClass - Classification in HAL domains"	"halClass - Classification dans les domaines HAL"
 "ws_post-v1-en-classhalen_description"	"Classification of documents in French or in English in the 13 major fields (in French) of the classification plan used by the HAL database."	"Classe des documents en français ou en anglais dans les  13 grands domaines (en français) du plan de classement utilisé par la base HAL."
-"ws_post-v2-unpaywall-expand_title"	"DOI enrichment via Unpaywall"	"Enrichissement par DOI via Unpaywall"
+"ws_post-v2-unpaywall-expand_title"	"doiEnrich Unpaywall - DOI enrichment via Unpaywall"	"doiEnrich Unpaywall - Enrichissement par DOI via Unpaywall"
 "ws_post-v2-unpaywall-expand_description"	"Retrieves the metadata of a scientific article in the Unpaywall database from a DOI."	"Récupère les métadonnées d’un article scientifique présent dans la base Unpaywall à partir d’un DOI."
-"ws_post-v2-openalex-expand_title"	"DOI enrichment via OpenAlex"	"Enrichissement par DOI via OpenAlex"
+"ws_post-v2-conditor-expand_title"	"doiEnrich Conditor - DOI enrichment via Conditor"	"doiEnrich Conditor -Enrichissement par DOI via Conditor"
+"ws_post-v2-conditor-expand_description"	"Retrieves the metadata of a scientific article in the Conditor database from a DOI."	"Récupère les métadonnées d’un article scientifique présent dans la base Conditor à 
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** ‏La chaîne de caractères du dernier champ n'est pas fermée!